lib/components/twitch/message.dart

```diff
@@ -77,6 +143,18 @@ Iterable<InlineSpan> parseText(String text, TextStyle linkStyle) {
         ),
       );
     } else {
+      var lst = tokenize(element.text);
+      print("tokens are: $lst");
+      // sample of lst: [[@rippyae, tag], [ fffff, regular]]
+      // how to return both TextSpan with a single widget;
```

Comment:
You can wrap multiple children in a `TextSpan`: https://api.flutter.dev/flutter/painting/TextSpan/children.html

So something like `return TextSpan(children: tokenize(element.text))` would work.